styles.css

```
@@ -57,29 +91,74 @@ body {
-    z-index: 1;
+    background: linear-gradient(135deg, #2c1654 0%, #341c6a 25%, #3a1b7a 50%, #4b1b8a 75%, #5a1b9a 100%);
+    z-index: 1; /* Lower z-index to be below cells */
     animation: gradient-shift 8s infinite alternate;
+    background-size: 100% 100%;
+    pointer-events: none; /* Allow clicks to pass through */
 }
 
 @keyframes gradient-shift {
     0% {
-        background: radial-gradient(circle at center, #2c1654 0%, #150b30 100%);
+        background: linear-gradient(135deg, #2c1654 0%, #341c6a 25%, #3a1b7a 50%, #4b1b8a 75%, #5a1b9a 100%);
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-size



(lint/suspicious/noShorthandPropertyOverrides)